pkg/local_workflows/local_workflows.go

```diff
@@ -5,14 +5,15 @@ import (
 )
 
 // Init initializes all local workflows
-// localworkflows are initialised when create a new workflow engine via app.CreateAppEngine()
+// localworkflows are initialized when create a new workflow engine via app.CreateAppEngine()
 func Init(engine workflow.Engine) error {
 	var err error
 
 	initMethods := []func(workflow.Engine) error{
 		InitDepGraphWorkflow,
 		InitOutputWorkflow,
 		InitWhoAmIWorkflow,
+		InitAuth, //Use legacy CLI for authentication for now, until OAuth is ready
```

Comment:
maybe add `TODO` to get a marker in the IDE

Reply:
This is a stale comment. The entire line was commented out